jpype/_jforward.py

```diff
@@ -168,33 +168,31 @@ def __init__(self, symbol):
         object.__setattr__(self, '_symbol', symbol)
         _jforward.append(self)
 
-    def __getattribute__(self, name):
-        forward = object.__getattribute__(self, '_symbol')
-        try:
-            return object.__getattribute__(self, name)
-        except AttributeError as ex:
-            if name.startswith("_"):
-                raise ex
-            if forward is None:
-                value = JForward(name)
-            elif name.startswith('['):
-                value = JForward(forward + name)
-            else:
-                value = JForward(forward + "." + name)
-            object.__setattr__(self, name, value)
-            return value
+    def __getattr__(self, name):
+        forward = self._symbol
+
```

Comment:
Caution - I removed the leading ``_`` case, since it raises. Might be the wrong thing to do.